### PR TITLE
Add for code review

### PR DESCRIPTION
**Pull request by Benawi**
## Code Review request

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <form class="m-auto px-2 d-flex position-relative">
+      <input id="add-task" class="w-100 p-2" type="text" placeholder="Add a task and press Enter key">
+        <div class="arrow-icon position-absolute"><i class="fas fa-2x fa-arrow-right"></i></div>
+      <img src="#" id="arrow-icon" class="icon arrow-icon position-absolute fas fa-2x fa-arrow-right" alt=""/>
+    </form>
+    <div id="todo-list-container" class="m-auto my-2 px-2">
+    </div>
+    <div>
+    <button id="clear-tasks" type="button" class="btn btn-light w-100 text-secondary">Clear all Tasks</button>
+    </div>
```
Comment (Peter-Dumbari) [inline src/index.html:26]: I suggest you use the `section` tag instead `div`, which will be more specific in showing the skeletal system of the page🤗

File: README.md [hunk 2]
```diff
@@ -1,41 +1,5 @@
 <a name="readme-top"></a>
```
Comment (Peter-Dumbari) [inline README.md:1]: You can also add an image like your app logo, to make the README file look more professional🤔

File: src/modules/UI.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        <input class="checkbox form-check-input" type="checkbox" id="blankCheckbox" value="true" aria-label="..." ${checked}>
+      </div>
+      <input id="text-input-${obj.index - 1}" class="text-description ${lineTrought}" type="text" placeholder="${obj.description}" value="${obj.description}"></input>
+      <img src="${EllipsisIcon}" class="icon ellipsis-icon fas fa-ellipsis-v ms-auto text-secondary"/>
+      <img src="${TrashIcon}" class="icon trash-icon ms-auto text-secundary d-none"/>
+    </div>`;
+    });
+    this.checkBOXHandler.addChangeListner();
+  }
+}
```
Comment (Peter-Dumbari) [inline src/modules/UI.js:43]: Your UI module is not containing a single delete and edit button, your app will look more professional if those functionalities are added 🤨.

**Review by Peter-Dumbari — COMMENTED** [review]
Great job so far implementing almost all the required functionalities👍.

## Review Suggestions ♻️ 
Please just take a look at the suggestion I made below and think about it. 
Happy coding 🤝.

File: src/index.html (hunk 1 above)
Comment (Benawi) [reply src/index.html:26]: Thank you, here I  have only one section, that is why used div.

Comment (Peter-Dumbari) [reply src/index.html:26]: Alright, but if you inspect the page with your browser, you will see the skeletal system am talking about, I still suggest you use `section` in place of `div`.

**Pull request merged**